cg/cli/workflow/balsamic/base.py

```diff
@@ -221,11 +228,14 @@ def store(context, case_id, analysis_type, dry):
 def start_available(context, dry):
     """Start full workflow for all available BALSAMIC cases"""
     balsamic_analysis_api = context.obj["BalsamicAnalysisAPI"]
+    exit_code = EXIT_SUCCESS
     for case_id in balsamic_analysis_api.get_cases_to_analyze():
         try:
             context.invoke(start, case_id=case_id, dry=dry)
         except click.Abort:
+            exit_code = EXIT_FAIL
             continue
+    sys.exit(exit_code)
```

Comment:
```suggestion
    if exit_code:
        raise click.Abort
```
If you use this you don't need sys